<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>351</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>1. Find Contour</string>
    </property>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>1.1 Draw Contour</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>1.2 Count Coins</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_coin1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>160</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>There are ___ coins in coin01.jpg</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_coin2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>190</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>There are ___ coins in coin02.jpg</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>351</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>2. Calibration</string>
    </property>
    <widget class="QPushButton" name="pushButton_21">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>2.1 Find Comes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_22">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>2.2 Find Intrinsic</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_24">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>2.4 Find Distortion</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>361</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>3. Augmented Reality</string>
    </property>
    <widget class="QPushButton" name="pushButton_31">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>27</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>3.1 Augmented Reality</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>130</y>
      <width>361</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>4. Stereo Disparity Map</string>
    </property>
    <widget class="QPushButton" name="pushButton_41">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>37</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>4.1 Stereo Disparity Map</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>280</y>
      <width>361</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>2.3 Find Extrinsic </string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select image</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_23">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>2.3 Find Extrinsic</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>965</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
